src/include/storage/store/var_list_column.h

```diff
@@ -25,19 +26,21 @@ namespace kuzu {
 namespace storage {
 
 struct ListOffsetInfoInStorage {
-    common::offset_t prevNodeListOffset;
+    common::offset_t numTotal;
     std::vector<std::unique_ptr<common::ValueVector>> offsetVectors;
+    std::vector<std::unique_ptr<common::ValueVector>> sizeVectors;
```

Comment:
You can rework `offsetVectors` and `sizeVectors` into ColumnChunk, instead of a vector of ValueVector. Using ColumnChunk should simplify code in this class.